narratives/ncov_sit-rep_2020-04-24.md

```diff
@@ -0,0 +1,259 @@
+<br><br>
+While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
+<br><br>
+For example, the map shows very few sequences from East Africa. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Many separate introductions have seeded diverse local transmission chains in Africa](https://nextstrain.org/ncov/africa/2020-04-23?d=tree,map&f_region=Africa&p=full&r=country&legend=open)
+  At the time of writing, the [Africa CDC](https://africacdc.org/covid-19/) reports 26,144 confirmed COVID-19 cases and 1,247 deaths across the African Union.
+  We have viral genomes from 83 of these cases from 7 African nations, shown here (in color) against a backdrop of contextual sequences from around the world (gray). We applaud the local public health departments for their rapid response and open data sharing.
+  <br><br>
+  These samples span the full breadth of the tree, indicating that there have been many independent introductions to the African continent.
+  Many of these introductions appear to be from Europe, although this may be skewed by the fact that we have more sequences from Europe than from most other places in the world; we can't rule out the possibility of unsampled, intermediate transmissions elsewhere.
+  <br><br>
+  Not all of these introductions have led to community transmission, but we see clear clusters in some locations.
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [At least 7 introductions to the DRC](https://nextstrain.org/ncov/africa/2020-04-23?d=tree,map&f_region=Africa&p=full&r=country&legend=closed&f_country=Democratic%20Republic%20of%20the%20Congo)
+
+There are 359 confirmed cases of COVID-19 in the DRC with 25 deaths. On Nextstrain, we have 36 sequences from the DRC.
+<br><br>
+These sequences fall across the tree, with evidence for at least 7 separate introductions. Heuristically, introductions look like a color change between a virus sample (tips of the tree) and its ancestors (the internal nodes or branch points that lead to it). From these 7 introductions, we see evidence for at least 2 local transmission chains.
```

Comment:
```suggestion
These sequences fall across the tree, with evidence for at least 7 separate introductions. As shown here, introductions look like a color change between a virus sample (tips of the tree) and its ancestors (the internal nodes or branch points that lead to it). 

Here, we can see 7 places where the colors have changed from grey to orange, representing an introduction. From these 7 introductions, we see evidence for at least 2 local transmission chains.
```
I think if we're trying to stick to more basic English, better to avoid 'heuristically' :)
For the second part, take or leave - thought it was nice to tie up your very nice explanation with exactly what's shown, but just a suggestion!